packages/yoastseo/src/worker/createWorker.js

```diff
@@ -37,12 +37,12 @@ function createBlobScript( url ) {
  * @returns {boolean} Whether the URLs have the same origin.
  */
 function isSameOrigin( urlA, urlB ) {
-	urlA = new URL( urlA, window.location.origin );
-	urlB = new URL( urlB, window.location.origin );
+	const url1 = new URL( urlA, window.location.origin );
+	const url2 = new URL( urlB, window.location.origin );
```

Comment:
This minor change prevents a warning in PhpStorm that the variable type is being changed from a string to a URL.